src/precice/impl/SolverInterfaceImpl.hpp

```diff
@@ -625,6 +625,15 @@ class SolverInterfaceImpl {
   /// Communicate meshes and create partitions
   void computePartitions();
 
+  /// Helper for mapWrittenData and mapReadData
+  void computeMappings(utils::ptr_vector<MappingContext> contexts, std::string mappingType);
+
+  /// Helper for mapWrittenData and mapReadData
+  void mapData(utils::ptr_vector<DataContext> contexts, std::string mappingType);
```

Comment:
I guess calling `mappingType` by const reference here should be sufficient, right?

Reply:
Yes. I'm always getting very confused, when passing strings. So thanks for giving me this hint :D Fixed in e359bae998115c471113d20a701dfea0feaac7a6.